src/common/components/atoms/TextComponent.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
+import styled from 'styled-components';
+
+const StyledTextComponent = styled.div`
+background-color: peachpuff;
```

Comment:
Can we add a tab here s'il vous plait? 👀

Reply:
Done!